lib/galaxy_test/api/test_workflows.py

```diff
@@ -2208,15 +2205,6 @@ def test_workflow_invocation_report_custom(self):
             assert "## Workflow Inputs" in markdown_content
             assert "## About This Report" in markdown_content
 
-    @skip_without_tool("cat1")
-    def test_export_invocation_bco(self):
```

Comment:
Should we also (temporarily) restore this test?

Reply:
Sure! I can do that

Reply:
Thanks!

Reply:
While at it, I decided to validate the whole BCO against their schema as part of the tests in [c776c81](https://github.com/galaxyproject/galaxy/pull/14620/commits/c776c8169a9488a891bc71f05a3fd7e45b90d9b5) and we can keep that for future tests.

Reply:
@davelopez 
How did that work out? The validation test? I know there are a few things that can not be pulled from Galaxy invocation and so it might always fail but if it is an expected validation fail that can still be a great test. Right?

Reply:
@HadleyKing, the schema validation using https://opensource.ieee.org/2791-object/ieee-2791-schema/raw/master/2791object.json seems to validate fine for the workflow invocations under test (at least with the changes included in this PR). John Chilton generated the pydantic models from the schema and thanks to that found a couple of errors in the format that should be fixed now here.

Maybe there is still missing information, but the format itself returned by the API complies with the schema now.